src/tables/ebdt.rs

```diff
@@ -0,0 +1,30 @@
+//! An [Embedded Bitmap Data Table](
```

Comment:
Since this is just a wrapper, we can use a typedef here. Just put it in CBDT and CBLC files respectively. No need for a new type.

Reply:
These are public types that are exposed by the API. If we use a typedef, the user can rely on the fact that they are the same types and it would be a breaking change to separate the implementations. For instance, if we found some place where they need to be handled different. Are you sure you want this?

Also I'm a bit confused about what exactly you're asking for. Do you want the EBDT table type to be in the `cbdt` module? And if I did put it there, what would it be called, since the name `Table` is already taken?

Reply:
Breaking changes are expected. This is not 1.0 library. No problem.

Oh yeah, typedef would not work. We need:

```rust
pub mod ebdt {
    pub use crate::cbdt::Table;
}
```

I thinks this should work.

Or maybe we can skip explicit EBDT+EBLC tables altogether? Can font have EBDT+EBLC and CBDT+CBLC at the same time? Probably not. In which case we can parse EBDT into CBDT and so on. And add a comment somewhere explaining the logic.

Reply:
That code wouldn't give us the right documentation, since it would still point to the CBDT specs.

My preference would be to keep things as is (which gives the correct documentation and doesn't expose implementation details). If not that, then quietly parse EBDT into CBDT as you proposed, but I have no idea whether a font can have both and if this is okay. Anyway, up to you.